test/rulesets/LinearAlgebra/structured.jl

```diff
@@ -7,6 +7,12 @@
         # Concrete type instead of UnionAll
         rrule_test(typeof(D), D, (randn(rng, N), randn(rng, N)))
     end
+    @testset "::Diagonal * ::AbstractVector" begin
+        rng, N = MersenneTwister(123456), 3
+        rrule_test(*, randn(rng, N),
+                   (Diagonal(randn(rng, N)), Diagonal(randn(rng, N))),
+                   (randn(rng, N), randn(rng, N)))
```

Comment:
For consistancy with the rest of the code, the intentation style is:
```
        rrule_test(
            *, randn(rng, N),
            (Diagonal(randn(rng, N)), Diagonal(randn(rng, N))),
            (randn(rng, N), randn(rng, N)),
        )
```

i.e. always start and end a multiline call with a line that has no arguments on it,
and always ident by 4, rather than indenting to line-up with the brace

Reply:
actually it would be with one argument per line
```julia
rrule_test(
    *,
    randn(rng, N),
    (Diagonal(randn(rng, N)), Diagonal(randn(rng, N))),
    (randn(rng, N), randn(rng, N)),
)
```

Reply:
Yes, but I feel like this is one of the cases where the clear relationship between 
`*, randn(rng, N),` as inputs to the function makes it worth putting them on one-line.

I'ld be happy with either

Reply:
Good to know that ChainRules.jl is using a sane code style!